src/contexts/contracts/collateral-bridge.ts

```diff
@@ -0,0 +1,80 @@
+import { addDecimal } from "@vegaprotocol/smart-contracts-sdk";
```

Comment:
chore (blocking): we need to use the old bridge, this is not going to mainnet as part of this release.